Aphrodis/MyDiary_Challenge
# Bg fix ui #169448234

**What does this PR do?**
Creates responsive UI pages

**Description of Task to be completed?**
- Decrease while spaces on sign up and sign-in pages
- Modify nav bar
- Create a new landing page
- Make changes on dashboard page
- Add styles to new entry, modify entry and profile pages

**How should this be manually tested?**
In your terminal or command line:
- Clone the repo using: ```git clone https://github.com/Aphrodis/MyDiary_Challenge.git```
- Enter in the folder by doing ```cd MyDiary_Challenge```
- Check out the branch by ```git checkout bg-fix-UI-#169448234```
- Run UI pages in the browser
**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
#169448234

**Screenshots**
![dashboard](https://user-images.githubusercontent.com/34691089/67864416-43702080-fb2e-11e9-9232-e067cafa04c0.PNG)
![modify entry](https://user-images.githubusercontent.com/34691089/67864425-4834d480-fb2e-11e9-90ab-2653ad8b46cb.PNG)
![new entry](https://user-images.githubusercontent.com/34691089/67864434-4a972e80-fb2e-11e9-80bc-9fcf53800976.PNG)
![profile](https://user-images.githubusercontent.com/34691089/67864450-4ff47900-fb2e-11e9-8072-4568cfac808d.PNG)
![signin](https://user-images.githubusercontent.com/34691089/67864457-53880000-fb2e-11e9-880a-f73fe2b3aa19.PNG)
![signup](https://user-images.githubusercontent.com/34691089/67864464-55ea5a00-fb2e-11e9-8da7-336a55fe66ae.PNG)
